Rakefile

```diff
@@ -1,91 +1,92 @@
-SWIFTLINT_VERSION="0.41.0"
-XCODE_WORKSPACE="Simplenote.xcworkspace"
-XCODE_SCHEME="Simplenote"
-XCODE_CONFIGURATION="Debug"
+# frozen_string_literal: true
+
+require 'English'
+SWIFTLINT_VERSION = '0.41.0'
+XCODE_WORKSPACE = 'Simplenote.xcworkspace'
+XCODE_SCHEME = 'Simplenote'
+XCODE_CONFIGURATION = 'Debug'
 
 require 'fileutils'
 require 'tmpdir'
 require 'rake/clean'
 require 'yaml'
 require 'digest'
-PROJECT_DIR = File.expand_path(File.dirname(__FILE__))
+PROJECT_DIR = __dir__
```

Comment:
Same suggestion as for [the iOS PR](https://github.com/Automattic/simplenote-ios/pull/1507), it would be nice to group the `require` together and the constants together.

Reply:
Fixed in https://github.com/Automattic/simplenote-macos/commit/ec029d5e14b585ae0f471a936391fa32426fe86a